### DISCUSSION TITLE
LCTT1.0.4 certification example：Additional DR Decay test for only DC plan devices which support the optional data rates

### DISCUSSION THREAD

**Discussion by engineer-lin** [Lora-net/LoRaMac-node#1081] [Q&A]
Hi:
When the test progressed to the tenth step, I ran into a problem.
TCL first sends the MAC instructions for adding DR6 channels and adding DR7 channels to the End Device, and then modifies its channelsmask to only run on the two newly added channels. But I feel confused, these two newly added channels do not have DR0 to DR5, how to decay from the new channel to 5 to 0.
My test case can only attenuate from DR7 to DR6, and then from DR6 to DR0, causing the test case to fail. 
Hope to get your answer. This is very useful to me.
![image](https://user-images.githubusercontent.com/44599451/114526468-99307e00-9c79-11eb-9bc3-24ac9964264c.png)

**Comment by mluis1** [comment]
The MAC layer automatically handles the data rate decay from any DR.

Yesterday, I have run the test using last released code (v4.5.1) and LCTT with v3.1.0_R1 technology package and everything went well.
The used platform was a NucleoL476+SX1261MBXBAS.
Please see attached the generated log

[TP_A_EU868_ED_MAC_104_BV_012_B.zip](https://github.com/Lora-net/LoRaMac-node/files/6310624/TP_A_EU868_ED_MAC_104_BV_012_B.zip)

In the future it would be nice if you could post this kind of questions on the project [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. It is a better place to engage discussions and then we can agree if it is an issue or not.
If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.
